apps/website/.vuepress/config.js

```diff
@@ -0,0 +1,85 @@
+const markdownItAttrs = require('markdown-it-attrs');
+const sidebar = require('./sidebar');
+const blocks = require('./blocks');
+const path = require('path');
+
+module.exports = {
+  title: 'Clarity Design System',
+  description: 'Clarity Design System is the premiere design system for application development.',
+  head: [['link', { rel: 'shortcut icon', href: '/favicon.ico' }]],
+  dest: '../../dist/website',
+  themeConfig: {
+    // algolia: {
+    //   apiKey: 'd3349af25a19e3824cb48e57fe88fdcb',
+    //   indexName: 'clarity',
+    // },
+    lastUpdated: false,
+    search: false, // Disabling for the moment
+    sidebar,
+  },
+  markdown: {
+    extendMarkdown: md => {
+      // Disabling include plugin until I figure out if it casuse Vuepress to hang.
+      // md.use(require('markdown-it-include'));
```

Comment:
Remove this line, and make sure its not in package since we don't use it anymore

Reply:
removed.